content-repo/extra-docs/releases/20.9.1.md

```diff
@@ -0,0 +1,1205 @@
+- Added the *return_results* function and the *CommandResults* class to the **Script Helper** documentation.
+- Enhanced the *dict_safe_get* function, which now supports a nested list/dictionary and returns the type validation.
+- Added the *build_number* argument to the *is_demisto_version_ge* function and adjusted the *is_versioned_context_available* function accordingly.
+- Added the timeline object to the *CommandResult* class.
+
+##### SaneDocReports  
+- Fixed a trend percentage calculation issue.
+- Maintenance and stability enhancements.
+
+##### New: GetIndicatorsByQuery  
+Returns indicators that match the defined query and exports them to a file.
+
+##### DBotMLFetchData  
+- Added the *Created* field to the collected features.
+- Maintenance and stability enhancements.
+
+##### DBotPredictPhishingWords  
+- Maintenance and stability enhancements.
+- Changed the name of the *labelProbabilityThreshold* argument to *confidenceThreshold*.
+- Changed the default value of the *run once* parameter to **true**.
+
+##### CommonServerPowerShell  
+Added the *TableToMarkdown* method.
+
+##### DBotTrainTextClassifierV2  
+Fixed a typo in an error message.
+
+##### DBotSuggestClassifierMapping  
+Maintenance and stability enhancements.
+
+##### SanePdfReports  
+Maintenance and stability enhancements.
+
+---
+
+### Blueliv ThreatCompass Pack v1.0.1
+#### Integrations
+##### Blueliv ThreatCompass  
+Documentation and metadata improvements.
+
+---
+
+### CVE Search Pack v1.0.2
+#### Scripts
+##### cveSearch  
+Maintenance and stability enhancements.
+
+##### cveLatest  
+Maintenance and stability enhancements.
+
+---
+
+### Cisco Threat Grid Pack v1.1.0
+#### Playbooks
+##### Detonate File - ThreatGrid  
+Samples are now submitted as private by default.
+
+---
+
+### Cofense Triage Pack v1.1.5 (Partner Supported)
+#### Integrations
+##### Cofense Triage (Deprecated)  
+Maintenance and stability enhancements.
+
+---
+
+### Common Playbooks Pack v1.8.0
+#### Playbooks
+##### Detonate File - Generic  
+Fixed a bug where a change in the **Detonate File - ThreatGrid** inputs did not populate into the **Detonate File - Generic** playbook. Detonate File - ThreatGrid now submits private samples by default.
+
+---
+
+### Common Scripts Pack v1.2.41
+#### Scripts
+##### ParseEmailFiles  
+- Fixed an issue where S/MIME .eml files were not parsed properly.
+- Fixed a bug in which a double period (`..`) in a new line was not processed correctly due to the SMTP standard.
+
+##### RunPollingCommand  
+Improved error handling.
+
+##### SetAndHandleEmpty  
+The *stringify* and *append* parameters now work as expected.
+
+##### LookupCSV  
+Added support for values that contain commas in CSV files.
+
+##### JSONtoCSV  
+Maintenance and stability enhancements.
+
+##### DeleteContext  
+Updated the script to execute using the DBot role.
+
+---
+
+### Common Types Pack v1.9.0
+
+
+---
+
+### Common Widgets Pack v1.0.1
+#### GetLargestInputsAndOuputsInIncidents
```

Comment:
What changed here? Is this new or updated?

Reply:
It's new, but only from version 6.0.0

Reply:
so you can remove it

Reply:
I deleted it already